Store.html

```diff
@@ -25,31 +25,30 @@
     </div>
     </div>
     <!-- End of Header  -->
+
+
 <!-- form and picture  -->
 <div class="container">
   
-        <img class="form-image" src="../HTML-CSS-Module- Karma-Project/level-2/store-image_by-andrew-neel-unsplash.jpg" alt="">
-
+        <img src="https://images.unsplash.com/photo-1535957998253-26ae1ef29506?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8b2ZmaWNlfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="store image">
```

Comment:
You should consider downloading the image rather than using the link because the image could be removed from the link could be removed from the web or wherever you got it from.